packages/buidler-core/src/builtin-tasks/node.ts

```diff
@@ -63,6 +72,66 @@ Private Key: ${privateKey}
 }
 
 export default function () {
+  internalTask(TASK_NODE_WATCH_COMPILER_OUTPUT)
+    .addParam("url", "The URL of the node", undefined, types.string)
+    .setAction(async ({ url }: { url: string }, { config }) => {
+      const { default: fetch } = await import("node-fetch");
+      const chokidar = await import("chokidar");
+
+      const compilerVersion = config.solc.version;
+      const solcInputPath = path.join(config.paths.cache, SOLC_INPUT_FILENAME);
+      const solcOutputPath = path.join(
+        config.paths.cache,
+        SOLC_OUTPUT_FILENAME
+      );
+
+      const addCompilationResult = async () => {
```

Comment:
What would happen if this throws?

Reply:
The node continues to work, but the raw error is shown in the console. I'll add a try/catch around it so that only a warning is shown.